frappe/tests/ui_test_helpers.py

```diff
@@ -138,11 +138,12 @@ def create_contact_records():
 def create_multiple_todo_records():
 	if frappe.db.get_all('ToDo', {'description': 'Multiple ToDo 1'}):
 		return
-	for index in range(501):
-		frappe.get_doc({
-			'doctype': 'ToDo',
-			'description': 'Multiple ToDo {}'.format(index+1)
-		}).insert()
+
+	query = "INSERT INTO `tabToDo` (`name`, `description`) VALUES ('1001', 'Multiple ToDo 1')"
+	for index in range(1000):
+		query = query + ", ('100{}', 'Multiple ToDo {}')".format(index+2,index+2)
+
+	frappe.db.sql(query)
```

Comment:
`frappe.db.bulk_insert` might be better here. (yes, it exists 😆)

Reply:
also lack of standard fields will likely break UI. 

So maybe just do `db_insert` instead?

Reply:
> `frappe.db.bulk_insert` might be better here. (yes, it exists 😆)

I didn't know cool 👍, it should be on the docs.

Reply:
> also lack of standard fields will likely break UI.
> 
> So maybe just do `db_insert` instead?

I didn't get it?

Reply:
If you're just doing raw SQL insert it won't populate many standard fields in the row like `modified`, `owner` etc. `db_insert` will do that.